app/src/main/java/fr/free/nrw/commons/quiz/QuizController.java

```diff
@@ -0,0 +1,50 @@
+package fr.free.nrw.commons.quiz;
+
+import android.content.Context;
+
+import java.util.ArrayList;
+
+import butterknife.BindView;
+import fr.free.nrw.commons.R;
+
+/**
+ * controls the quiz in the Activity
+ */
+public class QuizController {
+
+    ArrayList<QuizQuestion> quiz = new ArrayList<QuizQuestion>();
+
+    public void initialize(Context context){
+        QuizQuestion q1 = new QuizQuestion(1,
+                context.getResources().getString(R.string.quiz_question_string),
+                "https://i.imgur.com/0fMYcpM.jpg",
```

Comment:
It would be nice to have these string urls at the top of this class as `private static final String`. 
- That way it would be much easier to change the URLs without navigating through the whole file. 
- The variable name would give some hint about the picture.